doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -8,6 +8,25 @@ Add video here
 
 Add text content
 
+
+### Training repo: Add your changes using the GitHub UI
+
+> *Fork the training repo (Additional information: The default branch of the training repo is the main branch. So on the "Create a new fork" page, make sure to uncheck the "Copy the main branch only" checkbox before clicking on the "create fork" button. This will allow you to also have the develop branch, since you will need the develop branch for the next steps).
+
+
+> *Go to the develop branch of your fork.
```

Comment:
This is not formatted as a list

Reply:
Thank you for the update ma. @Ifycode  I have changed it to a list.